package.json

```diff
@@ -45,7 +49,7 @@
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^5.43.0",
     "@typescript-eslint/parser": "^5.43.0",
-    "eslint": "^8.27.0",
+    "eslint": "8.22.0",
```

Comment:
Please update `eslint` to the latest version

Reply:
Done

Reply:
Done